supertokens_python/recipe/thirdpartypasswordless/asyncio/__init__.py

```diff
@@ -37,55 +39,67 @@ async def get_user_by_id(
 async def get_user_by_third_party_info(
     third_party_id: str,
     third_party_user_id: str,
+    tenant_id: Optional[str],
```

Comment:
need to make it as None.. check all places

Reply:
Okay